frontend/src/components/Pages/InspectionPage/InspectionTable.tsx

```diff
@@ -22,161 +22,360 @@ const StyledIcon = styled(Icon)`
+                        {TranslateText('No planned inspection')}
+                    </StyledContent>
+                )
             }
+            lastCompleted = TranslateText('Never')
+        } else {
+            status = inspection.missionDefinition.inspectionFrequency ? (
+                getInspectionStatus(inspection.deadline!)
+            ) : (
+                <StyledContent>
+                    <StyledCircle style={{ background: 'green' }} />
+                    {TranslateText('No planned inspection')}
+                </StyledContent>
+            )
+            lastCompleted = formatDateString(mission.lastRun.endTime!)
         }
+    }
 
-        return (
-            <Table.Row key={mission.id}>
-                <Table.Cell>{status}</Table.Cell>
-                <Table.Cell>
-                    <Typography
-                        link
-                        onClick={() => navigate(`${config.FRONTEND_BASE_ROUTE}/mission-definition/${mission.id}`)}
-                    >
-                        {mission.name}
-                    </Typography>
-                </Table.Cell>
-                <Table.Cell>{mission.area.areaName}</Table.Cell>
-                <Table.Cell>{mission.comment}</Table.Cell>
-                <Table.Cell>{lastCompleted}</Table.Cell>
-                <Table.Cell>{inspection.deadline ? inspection.deadline.toDateString() : ''}</Table.Cell>
-                <Table.Cell>
-                    <StyledIcon
-                        color={`${tokens.colors.interactive.focus.hex}`}
-                        name={Icons.AddOutlined}
-                        size={16}
-                        title={TranslateText('Add to queue')}
+    return (
+        <Table.Row key={mission.id}>
+            <Table.Cell>{status}</Table.Cell>
+            <Table.Cell>
+                <Typography
+                    link
+                    onClick={() => navigate(`${config.FRONTEND_BASE_ROUTE}/mission-definition/${mission.id}`)}
+                >
+                    {mission.name}
+                </Typography>
+            </Table.Cell>
+            <Table.Cell>{mission.comment}</Table.Cell>
+            <Table.Cell>{mission.area.areaName}</Table.Cell>
+            <Table.Cell>{lastCompleted}</Table.Cell>
+            <Table.Cell>{inspection.deadline ? formatDateString(inspection.deadline.toISOString()) : ''}</Table.Cell>
+            <Table.Cell>
+                {!isScheduled && (
+                    <Button
+                        variant="ghost_icon"
                         onClick={() => {
                             openDialog()
-                            setSelectedMissions([mission])
+                            setMissions([mission])
                         }}
-                    />
-                </Table.Cell>
-            </Table.Row>
-        )
+                    >
+                        <StyledIcon
+                            color={`${tokens.colors.interactive.focus.hex}`}
```

Comment:
In inspectionSection line 321 you use rgba while here its hex. Is there a reason why both is used?

Reply:
No, will change